coralproject/talk
# [CORL-2467] Make mobile toolbar's `Unmark all` work with server side seen comments

## What does this PR do?

- Removes the giant indexed db seen comments provider
- Hooks all the old removed functionality into the new seen comments mutations/queries
- Updates the mutations for marking seen comments to handle multiple comments
- Hooks the keyboard and mobile toolbar actions into the updated mutations

## What changes to the GraphQL/Database Schema does this PR introduce?

Updates `MarkCommentSeenPayload` and `MarkCommentSeenInput` to handle multiple comments/id's.

## Does this PR introduce any new environment variables or feature flags? 

No.

## How do I test this PR?

- Enable the comment seen functionality.
- Setup your  environment to test on mobile (or deploy to a test environment on GKE)
- Make some comments with one user
- On your phone, view that comment stream as a different user
- See that the comments are yellow for un-seen
- Press `Unmark all` in the toolbar
- Comments should be marked seen (go white instead of yellow)
- Test other z-key functionality to make sure it isn't now broken from the changes
 
## How do we deploy this PR?

No special considerations.
